airbyte-integrations/connectors/source-asana/integration_tests/sample_config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this should be a sample config

Reply:
What does this mean? Like we can't put actual `access_token` there. For now I'll change content of this file to this:
```
{
  "access_token": "TODO"
}
```